blocks/hero/hero.css

```diff
@@ -31,24 +59,99 @@ main .hero picture {
   box-sizing: border-box;
 }
 
+main .hero > div:first-of-type p > picture {
+  display: none;
+}
+
+main .hero > div:first-of-type p:last-of-type > picture {
+  display: block;
+}
+
 main .hero img {
   object-fit: cover;
   width: 100%;
   height: 100%;
 }
 
-main .hero > div > div:nth-of-type(2) {
-  max-width: 1200px;
+main .hero > div:nth-of-type(n+2) {
+  max-width: 90%;
   margin-left: auto;
   margin-right: auto;
+  display: flex;
+}
+
+main .hero > div:nth-of-type(n+2) div:nth-of-type(2) {
+  flex-basis: 8.5rem;
+  margin-left: auto;
+  transform: translateY(-1.5625rem);
+}
+
+main .hero > div:nth-of-type(n+2) div:nth-of-type(n+3) {
+  display: none;
 }
 
-@media (min-width: 900px) {
+main .hero > div:nth-of-type(n+2) >div:first-of-type:last-of-type {
+  max-width: 67%;
+}
+
+main .hero > div:nth-of-type(n+2) div:nth-of-type(1):not(:last-of-type) {
+  flex-basis: 42%;
+}
+
+main .hero > div:nth-of-type(n+2) div:nth-of-type(2) p:nth-of-type(n+2) > picture {
+  display: none;
+}
+
+@media (min-width: 56.25rem) {
```

Comment:
one question about the media queries, do we change them to rem?
I was thinking that if we use the px, it would be easier to identify the screen size